packages/block-editor/src/components/block-settings-menu-controls/index.js

```diff
@@ -37,12 +37,16 @@ const BlockSettingsMenuControlsSlot = ( { fillProps, clientIds = null } ) => {
 	return (
 		<Slot fillProps={ { ...fillProps, selectedBlocks } }>
 			{ ( fills ) => {
-				return (
-					<MenuGroup>
-						{ fills }
-						<ConvertToGroupButton onClose={ fillProps?.onClose } />
-					</MenuGroup>
-				);
+				if ( fills.length > 0 ) {
```

Comment:
Good addition 👍

We should account for the case when fills isn't an array.

We use `isEmpty` from lodash in other places but it should be fine to go with the [optional cgaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here:
```suggestion
				if ( fills?.length > 0 ) {
```

Reply:
Nice! Such a good learning experience, thank you.